spec/e2e/id_property_spec.rb

```diff
@@ -64,7 +64,7 @@
           Neo4j::Config[:id_property] = :the_id
           Neo4j::Config[:id_property_type] = :auto
           Neo4j::Config[:id_property_type_value] = :uuid
-          stub_const('Clazz', Class.new do
+          stub_const('Clazz', UniqueClass.create do
```

Comment:
If you don't want to do this in this PR I would be just as happy with a quick issue covering it